### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by Prime-5**
Updated README as listed in #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
 
-Understanding Kotlin concepts like OOPs.
+This repository is to help beginners understand and implement basic Kotlin concepts (like OOPS).
 
-For contribution-
-Add a Kotlin file that explains basic concepts of programming like -
+## Ideas for Contribution
 
-Some ideas from the top of my head.
+* Programs demonstratig how to use Arrays, Lists, and other data structures.
```
Comment (calra123) [inline README.md:7]: Fix the typo

File: README.md [hunk 2]
```diff
@@ -1,27 +1,46 @@
...
-`git push origin your_branch_name`
+### Fork this repository
+* Click on the "Fork" button on the top right corner of this repository to create its copy in your account.
 
+### Clone the repo to your machine
+* Go to your GitHub account and open the forked repository. 
+* Click on "Clone or Download" (A green button alongside Create new file/ Upload files/ Find file)
+* Copy the URL.
+* Open a terminal/git on your machine and run the following command (without <,>)
+`git clone <url_that_you_just_copied>`
```
Comment (calra123) [inline README.md:24]: Show it in the next line

**Review by calra123 — COMMENTED** [review]
Make the following changes

**Comment by Prime-5** [comment]
I have corrected the said errors in #4

**Pull request merged**